packages/react-sdk/docusaurus/docs/React/02-guides/10-sorting-api.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Participant sorting
+description: Overview of the Sorting API used to sort call participants.
+---
+
+## Introduction
+
+The Participant Sorting API is a powerful tool built on top of the internal `Comparator<T>` API,
```

Comment:
Would it make sense to add link to the resource that elaborates more on what Comparator API is?

Reply:
Aha, I see that it is explained below